designs/service_function_chaining/SFC_design.md

```diff
@@ -133,32 +156,111 @@ Proposed API Definition
 ```
 ElementIdInfo registerInspectionPort(InspectionPortElement inspectionPort) throws NetworkPortNotFoundException, Exception;
 ```
+###### installInspectionHook()
+This API uses updated `InspectionPortElement` . This interfaces parentId argument carries either NULL in case of no port chain is created at target Openstack nodes or an already existing port chain Id to add more flow classifiers to that port chain ID. And the same parentId is used for `installInspectionHookId`. Detail installInspectionHook work flow is show below. `networkElement` will be a list of elements.  

```

Comment:
>interfaces parentId argument carries either NULL
where does null make sense? This should result in an error right? We are expecting the port chain to be created before the install inspection hook gets called